Makefile

```diff
@@ -304,50 +304,51 @@ endif
 
 # The configuration file
 
+SUBST=-e 's|%%$1%%|$(subst \,\\,$($1))|'
 utils/config.ml: utils/config.mlp config/Makefile
-	sed -e 's|%%AFL_INSTRUMENT%%|$(AFL_INSTRUMENT)|' \
-	    -e 's|%%ARCH%%|$(ARCH)|' \
-	    -e 's|%%ARCMD%%|$(ARCMD)|' \
-	    -e 's|%%ASM%%|$(ASM)|' \
-	    -e 's|%%ASM_CFI_SUPPORTED%%|$(ASM_CFI_SUPPORTED)|' \
-	    -e 's|%%BYTECCLIBS%%|$(BYTECCLIBS)|' \
-	    -e 's|%%BYTERUN%%|$(BYTERUN)|' \
-	    -e 's|%%CC%%|$(CC)|' \
-	    -e 's|%%CCOMPTYPE%%|$(CCOMPTYPE)|' \
-	    -e 's|%%CC_PROFILE%%|$(CC_PROFILE)|' \
-	    -e 's|%%EXT_ASM%%|$(EXT_ASM)|' \
-	    -e 's|%%EXT_DLL%%|$(EXT_DLL)|' \
+	sed $(call SUBST,AFL_INSTRUMENT) \
+	    $(call SUBST,ARCH) \
+	    $(call SUBST,ARCMD) \
+	    $(call SUBST,ASM) \
+	    $(call SUBST,ASM_CFI_SUPPORTED) \
+	    $(call SUBST,BYTECCLIBS) \
+	    $(call SUBST,BYTERUN) \
+	    $(call SUBST,CC) \
+	    $(call SUBST,CCOMPTYPE) \
+	    $(call SUBST,CC_PROFILE) \
+	    $(call SUBST,EXT_ASM) \
+	    $(call SUBST,EXT_DLL) \
 	    -e 's|%%EXT_EXE%%|$(EXE)|' \
```

Comment:
It's the only one where the substitution variable name doesn't match the Makefile variable name... I guess `utils/config.mlp` could be changed...